draft-regext-rdap-extensions.md

```diff
@@ -249,13 +228,9 @@ would be like so:
 While [@!RFC9082] describes the extension identifier as a prepended
 string to a path segment, it does not describe the usage of the
 extension identifier as a path segment which may have child path
-segments. This document updates [@!RFC9082] to allow the usage of
-extension identifiers as path segments which may have child path
-segments. For example, if the "foobar" extension defines the child
-paths "fizz" and "fazz", the URLs for this extension would be like so:
-
-    https://base.example/foobar/fizz
-    https://base.example/foobar/fazz
+segments. This document updates [@!RFC9082] in the following manner:
+bare extension identifiers (see (#bare_extensions)) MUST NOT be the 
+most siginificant path segment in an RDAP URL.
```

Comment:
siginificant -> significant

This is better but there is no follow-up guidance on "child path segments" in the previous sentence.